spec/controllers/v0/benefits_claims_controller_spec.rb

```diff
@@ -135,14 +135,16 @@
                 { message_type: 'lh.cst.evidence_requests',
                   claim_id: '600383363',
                   tracked_item_id: 395_084,
-                  tracked_item_type: 'Request 1' })
+                  tracked_item_type: 'Request 1',
+                  tracked_item_status: 'NEEDED_FROM_YOU' })
         expect(Rails.logger)
           .to have_received(:info)
           .with('Evidence Request Types',
                 { message_type: 'lh.cst.evidence_requests',
                   claim_id: '600383363',
                   tracked_item_id: 394_443,
-                  tracked_item_type: 'Submit buddy statement(s)' })
+                  tracked_item_type: 'Submit buddy statement(s)',
+                  tracked_item_status: 'NEEDED_FROM_YOU' })
```

Comment:
For testing, it doesn't really matter what the value of this status is, but I changed it to be more reflective of what it seems like this data would be. We'll be handling the specific types on the DataDog side.